packages/kbn-dev-utils/package.json

```diff
@@ -1,26 +1,25 @@
 {
   "name": "@kbn/dev-utils",
   "main": "./target/index.js",
-  "types": "./index.d.ts",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "babel src --out-dir target --copy-files",
-    "kbn:bootstrap": "yarn build --quiet",
+    "build": "tsc",
+    "kbn:bootstrap": "yarn build",
     "kbn:watch": "yarn build --watch"
   },
   "dependencies": {
     "chalk": "^2.4.1",
     "execa": "^1.0.0",
+    "exit-hook": "^2.2.0",
```

Comment:
Hey Spencer, why do we need this dep in this package.json and the root package.json?

Reply:
We need it here because it's used by the proc runner module, and in the root because it's used by src/dev/run. Those are two separate packages and should list their dependencies separately.